packages/valko-ui-components/src/types/Datepicker.ts

```diff
@@ -0,0 +1,7 @@
+import type { DefaultComponent } from './common'
+
+export type viewType = 'days' | 'months' | 'years'
```

Comment:
name is not consistent with  other type names, should be something like `DisplayView`